### ISSUE TITLE
0.7.0 is failing on my plugins

### ISSUE THREAD

**Issue by olivernybroe** [JetBrains/gradle-intellij-plugin#603]
I am getting the following errors in my plugins with the new version. 

```
Cannot create class com.intellij.application.options.editor.GutterIconsConfigurable
```

Not really sure what is causing it, I saw this error because of dependabot.
Here are links to my failing tests in CI 🙂

- https://github.com/pestphp/pest-intellij/pull/101
- https://github.com/olivernybroe/collector-intellij/pull/50

**Comment by hsz** [comment]
Thanks for reporting. The issue is caused by:

```
Caused by: java.lang.UnsatisfiedLinkError: /home/runner/.gradle/caches/modules-2/files-2.1/com.jetbrains/jbre/jbr-11_0_9_1-linux-x86-b1257.1/jbr/lib/libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
102
```

**Comment by olivernybroe** [comment]
@hsz Ah, thanks! 👍
Good to know that this is indeed a bug from this package.

**Comment by hsz** [comment]
The issue is fixed and released with v0.7.2.
